src/main/kotlin/Archive.kt

```diff
@@ -0,0 +1,51 @@
+﻿class Archive (val name: String) {
+    var noteList: MutableList<Note> = mutableListOf()
+    var exit: Boolean = false
+
+    fun menu(){
+        println(cuteCat + "====================\nАрхив \"$name\" \n====================")
+        println("1 - Создать новую заметку\n2 - Посмотреть существующую заметку\n3 - Нарисовать котика\n4 - Выход в меню архивов\n====================")
+        this.seeAllNotes()
+        var userInput: Int = 0
+        exit = false
+        userInput = enterNumberBetween(1,4)
```

Comment:
Не думаю, что нужно создавать число, а потом переприсваивать ему значение
Можно сделать проще
```suggestion
        val userInput = enterNumberBetween(1, 4)
```